tests/test_core.py

```diff
@@ -165,10 +165,9 @@ def test_marc21_field_247_matching():
         '247__': [{'a': 'B'}],
     })
 
-    assert data['other_standard_identifier'][0]['standard_number_or_code'] \
-        == 'A'
-    assert data['former_title'][0]['title'] == 'B'
-    assert len(data) == 2
+    assert 'A' == data['other_standard_identifier']['standard_number_or_code']
```

Comment:
the `GroupableOrderedDict` doesn't know jack about MARC and behaves the same way in all situations.